src/main/java/by/byshnev/dto/MapperArticleDTO.java

```diff
@@ -0,0 +1,14 @@
+package by.byshnev.dto;
+
+import by.byshnev.models.Article;
+import org.modelmapper.ModelMapper;
+
+public class MapperArticleDTO {
```

Comment:
what is it? how to use it?
if you want library that automatically creates mapper implementations, google for mapstruct